tests/ga/test_extension.py

```diff
@@ -601,16 +608,15 @@ def test_ext_handler_sequencing(self, *args):
                                     expected_handler_name="OSTCExtensions.OtherExampleHandlerLinux")
 
         # Test goal state changed
-        test_data.goal_state = test_data.goal_state.replace("<Incarnation>1<",
-                                                            "<Incarnation>2<")
-        test_data.ext_conf = test_data.ext_conf.replace("seqNo=\"0\"",
-                                                        "seqNo=\"1\"")
+        test_data.set_incarnation(2)
+        test_data.set_extensions_config_sequence_number(1)
         # Swap the dependency ordering
-        test_data.ext_conf = test_data.ext_conf.replace("dependencyLevel=\"2\"",
-                                                        "dependencyLevel=\"3\"")
-        test_data.ext_conf = test_data.ext_conf.replace("dependencyLevel=\"1\"",
-                                                        "dependencyLevel=\"4\"")
+        test_data.ext_conf = test_data.ext_conf.replace("dependencyLevel=\"2\"", "dependencyLevel=\"3\"")
```

Comment:
I didn't create utility functions for the dependency level, since it requires more sophisticated parsing of the xml... maybe next time

Reply:
We can't have it all...